python/lsst/ts/eas/eas_csc.py

```diff
@@ -84,290 +88,238 @@ def __init__(
             simulation_mode=simulation_mode,
             override=override,
         )
-        self.eas = None
-        self.m1m3ts = salobj.Remote(self.domain, "MTM1M3TS")
-        self.ess = salobj.Remote(self.domain, "ESS", index=112)
 
-        # Variables for the m1m3ts loop
-        self.m1m3_thermal_task = utils.make_done_future()
-        self.enabled_event = (
-            asyncio.Event()
-        )  # An event that is set when the CSC is enabled.
-        self.control_loop_lock = asyncio.Lock()
-
-        self.heater_demand: list[int] = [0] * 96
-        self.fan_demand: list[int] = [30] * 96
-        self.temperature_target_offset = -1.0
+        self.last_vec04_time: float = (
+            0  # Last time VEC-04 was changed (UNIX TAI seconds).
+        )
+        self.health_monitor_task = utils.make_done_future()
+        self.wind_history = pd.DataFrame(
```

Comment:
I am not sure this is good use of pandas data frame. Why not stick to regular list or numpy array? You could even consider a `collections.deque` which is a sized array and would prevent memory leaks.